e2e-tests/cypress/tests/integration/channels/test_notification/trigger_browser_notification_spec.ts

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @channels @notification
+
+describe('Verify users can receive notification on browser', () => {
+    let offTopic: string;
+    const notificationMessage = 'If you received this test notification, it worked!';
+    before(() => {
+        cy.apiInitSetup({userPrefix: 'other', loginAfter: true}).then(({offTopicUrl}) => {
+            offTopic = offTopicUrl;
+        });
+    });
+
+    it('MM-T5631 should be able to receive notification when notifications are enabled on the browser', () => {
+        cy.visit(offTopic);
+        cy.stubNotificationPermission('granted');
+        triggertestNotification();
+        cy.get('@notificationStub').should('be.called');
+        cy.get('@notificationStub').should((stub) => {
+            expect(stub).to.have.been.calledWithMatch(
+                'Direct Message',
+                Cypress.sinon.match({
+                    body: '@@system-bot: If you received this test notification, it worked!',
+                    tag: '@@system-bot: If you received this test notification, it worked!',
+                    requireInteraction: false,
+                    silent: false,
+                }),
+            );
+        });
+        cy.get('#accountSettingsHeader button.close').click();
+        cy.verifySystemBotMessageRecieved(notificationMessage);
+    });
+
+    it('should not be able to receive notification when notifications are denied on the browser', () => {
```

Comment:
Thanks for adding MM-T5631! How about his and the next test cases below?

Reply:
Sure Saturn. I'll add MM-T5631_1 as scenarios under 1 test case